docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -161,6 +161,9 @@ values. Typical usage does not require you to configure them.
   - Maximum number of metastore data objects per transaction in the Hive
     metastore cache.
   - `1000`
+* - `delta.cache-table-metadata`
```

Comment:
Do you have any concerns about potential clashes between this config name and `flush_metadata_cache` procedure name?

Reply:
How about `delta.metastore.cache-table-metadata`?

Reply:
`cache` is rather abstract so misleading.

Maybe `delta.metastore.store-table-metadata`